environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentLoadBalancerService.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.environment.environment.service;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.environment.environment.dto.EnvironmentDto;
+import com.sequenceiq.environment.environment.dto.EnvironmentLoadBalancerDto;
+
+@Service
+public class EnvironmentLoadBalancerService {
+
+    public void updateLoadBalancerInEnvironmentAndStacks(EnvironmentDto environmentDto, EnvironmentLoadBalancerDto environmentLbDto) {
+        // TODO
```

Comment:
something missing here ?

Reply:
No. This PR is just to stub these things out, and they're filled out in https://github.com/hortonworks/cloudbreak/pull/9963.